### ISSUE TITLE
Problem with validation script/regex module

### ISSUE THREAD

**Issue by vaporizedcamel** [UniversalDependencies/docs#577]
Dear All,

As I'm fixing the issues in the Akkadian treebank, it would be convenient for me to validate the file on my own to hunt for all of the invalid lines. But for some reason I can't get to work the regex module in python 2.7 to run the script properly. I've tried some workarounds - including implementing the regex module from the python wheel, but the validate.py script is not running in any configuration that I tried and ignores regex installed from the wheel.

Pip says:

> _Could not fetch URL https://pypi.python.org/simple/regex/: There was a problem confirming the ssl certificate: [Errno 1] _ssl.c:503: error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version - skipping_

And it seems that I can't successfully install any module into python 2.7

I am new to python and this might be something obvious, but I have no idea how to solve it.
Is there any solution to that problem that you know of?

**Comment by dan-zeman** [comment]
I am no expert in Python but this sounds to me as a problem that is specific to the communication between your computer and the server with Python modules, not to Python as such or to the validation script.

The regex module should be needed only to check the DEPS column, so if you have always an underscore there, and if you cannot make the standard validation script work, you should be able to run it after you uncomment the tenth line and comment out the eleventh:

<pre>
import re
#import regex as re
</pre>

**Comment by vaporizedcamel** [comment]
Thank you for your answer. Although I couldn't get the script to work, I circumvented the problem by refreshing the validation provided online. As much as it wasn't efficient, I fixed the issues.

One last question about the validation:

> Sat Oct 20 02:44:04 CEST 2018
> tools/check_files.pl UD_Akkadian-PISANDUB
> UD_Akkadian-PISANDUB extra files: error.log
> *** FAILED ***
> ./validate.sh --lang akk --max-err=10 UD_Akkadian-PISANDUB/akk_pisandub-ud-test.conllu
> [Line                   2210]: Only the unix-style LF line terminator is allowed
> *** FAILED *** with 1 errors
> Format errors: 1

What should be inserted as the line terminator at the end of the document? (I'm working on Windows, so that might be a problem)

**Comment by martinpopel** [comment]
Windows usually use CRLF terminators by default, but most text editors and most [programming languages](https://stackoverflow.com/questions/2536545/how-to-write-unix-end-of-line-characters-in-windows-using-python) allow to use LF line terminators.

**Comment by bulbulistan** [comment]
That’s not it, my files are CRLF and everything works fine.
I bet the actual problem is the extra file, fix that, the other one will go
away.

El ds, 20 oct 2018 a les 9:40 Martin Popel <notifications@github.com> va
escriure:

**Comment by dan-zeman** [comment]
@bulbulistan : If you use CRLF in your editor, it can still work fine if your git client converts it to LF when uploading the files. Some clients are configured to work that way by default. Anyways, CRLF really is illegal in CoNLL-U files.

@vaporizedcamel : I have fixed the file for you. (FYI: I did it on Windows, too. I use "Notepad2" which allows you to switch between CRLF and LF.)

Congratulations and thanks for preparing the data for release!
